src/main/java/de/epiceric/shopchest/listeners/ShopUpdateListener.java

```diff
@@ -23,48 +14,6 @@ public ShopUpdateListener(ShopChest plugin) {
         this.plugin = plugin;
     }
 
-    @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
-    public void onPlayerTeleport(final PlayerTeleportEvent e) {
```

Comment:
Didn't know about that, re-added [here](https://github.com/EpicEricEE/ShopChest/pull/131/commits/08b298922e75fb61cc32db7cb69c1d281a6e5125#diff-2cd426fb62a5b3a4227b16384e1e472bR24).